01_Data/src/layers/sequelize/model/CallMessage.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2023 S44, LLC
+// Copyright Contributors to the CitrineOS Project
+//
+// SPDX-License-Identifier: Apache 2.0
+
+import { Column, DataType, Index, Model, Table } from 'sequelize-typescript';
+
+@Table
+export class CallMessage extends Model {
+  static readonly MODEL_NAME: string = 'CallMessage';
+
+  @Index
+  @Column({
+    type: DataType.STRING,
+    unique: true,
+  })
+  declare correlationId: string;
+
+  @Column(DataType.INTEGER)
+  declare reservationId?: number;
```

Comment:
Why make this optional? If I understand this correctly, the only point is to be able to find the reservationId via correlationId

Reply:
Hi Christian, thanks for reviewing! 
My idea is that we may extend this table to use it for other models in the same way as well, e.g, adding a optional `chargingProfileId` col. In this case, reservationId should be optional.

Reply:
Ah got it. Would you mind adding a comment on your intent for the class? That it will be extended for other correlationId related lookups?